build.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
```

Comment:
What happens if the user doesn't have cuda libs installed or doesn't want to install them?

Reply:
Seems like we can have something like: `if env::var("CARGO_CFG_cuda").unwrap() { prints... }`

if we have a cfg like this:
`#[cfg(cuda)] => CARGO_CFG_cuda`